authorization/README.md

```diff
@@ -0,0 +1,35 @@
+# Tapsi Pack API: Authorization Guide
+
+Welcome to the Authorization guide for the Tapsi Pack API. This guide will help you understand how to authenticate with the Tapsi Pack API using OAuth and API Secret Key.
+
+## Overview
+
+In this document, external teams integrating with our API are referred to as **clients**.
+
+The Tapsi Pack Authorization process is handled by two different methods:
+- **OAuth**: This method is used to authenticate **users** and generate access tokens.
+- **API Secret Key**: This method is used to authenticate **clients**.
+
+### OAuth
+
+OAuth is a more complex, but also more secure and flexible method of authentication. 
```

Comment:
Why do we need to say that? since there are different usages and scenarios to decide between OAuth or Secret Key.

Reply:
changed it to `OAuth is used in a scenario in which the client wants to access the Tapsi Pack API on behalf of a user.`